src/omv/sensors/ov7670.c

```diff
@@ -21,6 +21,10 @@
 #include "ov7670_regs.h"
 #include "py/mphal.h"
 
+#define OMV_OV7670_CLKRC 0x00
+#define OMV_OV7670_VERSION 70
+
+
```

Comment:
```suggestion
```
This should be in the omv_boardconfig.h. Please copy from here: https://github.com/openmv/openmv/blob/master/src/omv/boards/NANO33/omv_boardconfig.h#L51